packages/utils/src/lib/reports/scoring.ts

```diff
@@ -51,7 +51,7 @@ export function scoreReport(report: Report): ScoredReport {
       )?.score;
       if (score == null) {
         throw new Error(
-          `Group has invalid ref - audit with slug ${slug}-${ref.slug}-audit not found`,
+          `Group has invalid ref - audit with slug ${ref.slug} from plugin ${slug} not found`,
```

Comment:
Note: Identical error message is also present in `perf` benchmarks. Not sure to which extent we want to maintain them though.